utils/route.tsx

```diff
@@ -2,14 +2,15 @@
 // See LICENSE.txt for license information.
 
 import {ErrorPageTypes} from 'utils/constants';
+
 import {isGuest} from 'utils/utils.jsx';
 
-export function importComponentSuccess(callback) {
-    return (comp) => callback(null, comp.default);
+export function importComponentSuccess(callback: any): any {
```

Comment:
Can we type the parameter and the return type here?
We can use something of the format `() => sometype`